kubeadm/flannel/Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG servercoreTag="ltsc2019"
+ARG flannelVersion="0.13.0"
 ARG cniVersion="0.8.5"
```

Comment:
There is a newer version of CNI, 0.8.7. Should any of these be defaulted? Or should these instead have no default value, and if no value is provided, just reach out to Github to ascertain the latest release version and use that instead? I did this in my own script module that I've successfully used to setup a Windows Server 2019 2004 server as a worker node for some of the dependencies, such as `yq` and `wins`. (I just haven't gotten around to "backporting" that to the other dependencies, such as Flannel and CNI in the docker images.)

Reply:
We could update this to .8.7

Reply:
Ok